src/components/page-types/formPage/tripCustomJourneys.js

```diff
@@ -46,11 +44,10 @@ const TripCustomJourneys = (props) => {
       >
         {heading}
       </Heading>
-      {hasRichText(body) && (
-        <RichTextRenderer
-          wysiwyg={body}
-          className="su-card-paragraph children:su-leading-snug children:!su-mb-06em children:last:!su-mb-0"
-        />
+      {body && (
+        <p className="su-intro-text children:su-leading-snug children:!su-mb-06em children:last:!su-mb-0">
```

Comment:
```suggestion
        <p className="su-intro-text su-rs-mb-5">
```
Same treatment as the other one